### ISSUE TITLE
0.92 pod error

### ISSUE THREAD

**Issue by wgywgy** [realm/realm-cocoa#1843]
#import realm/link_view.hpp
#import realm/binary_data.hpp
file not found
cocoapod version 0.37
![2015-05-06 13 52 37](https://cloud.githubusercontent.com/assets/2408246/7488099/50a410ec-f3f7-11e4-92f6-15aad9a8d860.png)

**Comment by jpsim** [comment]
Hi @wgywgy, I've just confirmed that `pod "Realm"` works with v0.92.0 in Xcode 6.3.1 both with and without `use_frameworks!` and both in Objective-C and Swift.

However, it looks like in your case, there was a problem downloading the core binary. Could you try running `rm -rf Pods/Realm/core`, then `pod install` again to see if it works? If it still doesn't work, please share the command line CocoaPods output here so we can help troubleshoot.

Also, please share which version of Xcode you're using and whether this is an Objective-C or Swift project.

**Comment by wgywgy** [comment]
xcode 6.3.1 and oc and without use_frameworks.
I delete the pods folder in the project and run pod install. However there was still error.

**Comment by mrackwitz** [comment]
CocoaPods caches downloaded dependencies since 0.37. Could you please try to nuke also the CocoaPods cache by running `rm -rf ~/Library/Caches/CocoaPods`? If there was an error on the first time when trying to download the core, this could still persit from there.

**Comment by nechtas** [comment]
I have the same issue. Deleting CocoaPods caches and reinstalling pods didn't help. Back to version 0.91.5 -> all works great.

**Comment by jpsim** [comment]
Hmm, the only change in the Podspec between those versions was to remove an unused `REALM_SWIFT` compiler flag (5b8bbceeb351236a51bd3a8a5368247fe2842978) and some minor updates to the summary, description, homepage, source url and documentation url (e6a7ecd4eca152e1193efb0bfcc9f92521098e5a). Could the switch from `https://github.com/Realm/realm-cocoa.git` to `https://github.com/realm/realm-cocoa.git` have caused this?

**Comment by segiddins** [comment]
@wgywgy out of curiosity, does this happen for you with CocoaPods v0.36.4?

**Comment by wgywgy** [comment]
run rm -rf ~/Library/Caches/CocoaPods 
also no use...
and I delete realm though pod file and add also no use.

**Comment by segiddins** [comment]
@wgywgy are you by change on a case-sensitive OS?

**Comment by mattyohe** [comment]
Similar issues here.

![screen shot 2015-05-12 at 10 49 53 pm](https://cloud.githubusercontent.com/assets/197446/7602943/3a3ebac6-f8f9-11e4-9eee-06fb8fee79f8.png)

```
[mattyohe:~/Work/test]$ cat Podfile.lock                                                                                                                                                   (master✱)
PODS:
  - Realm (0.92.2):
    - Realm/Headers (= 0.92.2)
  - Realm/Headers (0.92.2)

DEPENDENCIES:
  - Realm

SPEC CHECKSUMS:
  Realm: fa8e8f7f788dd63744485c15b453487bc7546c8e

COCOAPODS: 0.37.1
```

Noticed that Pods/Headers is completely empty on 92.2. So then I tried again, rm -rf'ing Pods/ and Podfile.lock and noticed it takes awhile to do a pod install. I realized I had been C-c'ing the install because I thought my internet had hung up. It just turns out I was on a slow connection and since realm core is "sizable" for a slow connection, it gave me the appearance of a timed out pod install. But, upon running pod install again, it nearly instantly installs, but I noticed things like the Pods/Headers directory was empty etc. It appears possible there's something wrong with Cocoapsds's caching?

Either way, no issue for me now with any version as of late.
